test/quantization/ao_migration/test_fuser_method_mappings.py

```diff
@@ -0,0 +1,34 @@
+from torch.testing._internal.common_utils import TestCase
+
+
+class TestFuserMethodMappingsPyImport(TestCase):
```

Comment:
I think so far all the tests have been put into the same file https://github.com/pytorch/pytorch/blob/master/test/quantization/ao_migration/test_quantize.py. We can probably rename the file name so it is more generic.

Reply:
ya thats been updated